routetable/route_table.go

```diff
@@ -122,6 +122,11 @@ func (r *RouteTable) SetRoutes(ifaceName string, targets []Target) {
 	r.dirtyIfaces.Add(ifaceName)
 }
 
+func (r *RouteTable) QueueResync() {
+	r.logCxt.Info("Queueing a resync.")
+	r.inSync = false
```

Comment:
Immediately, but `QueueResync` will only be called once every 30s.  

A community member put up a PR to monitor routes, which would improve that further: https://github.com/projectcalico/felix/pull/1316  but I see value in a periodic resync too; it's to clear to me that the netlink API is race-proof so I'd like to put this patch in too.